src/utils.js

```diff
@@ -3,6 +3,14 @@ const fs = require('fs');
 
 const { env } = require('./env.js');
 
+if (global.ReadableStream  === undefined && typeof process !== 'undefined') {
```

Comment:
Does this work in other environments with process defined? Like electron?

Reply:
I'm not sure about whether this would work under electron - since the require is performed in a try/catch it shouldn't cause a failure if for some reason the package is not available.

Reply:
I just did a quick test of a simple electron app and everything worked fine.

Reply:
Sweet!